### ISSUE TITLE
[StratCon] FAQ overhaul

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#2612]
I've augmented the existing StratCon FAQ with more info and also tricks and hints I gathered while playing.

I'll just copy-paste below rather than attaching, as it is plaintext.

```
StratCon reference and FAQ (v2.3)

============
BASICS AND SETUP
============

* To activate StratCon, go to the AtB tab in Campaign Options and check "Use StratCon" at the very bottom.
* The AtB Campaign State tab will be active. It will only have meaningful data once you arrive on the contract planet and the contract begins.
* Most AtB scenario options in the Campaign Options have no longer any effect when StratCon is enabled (namely: intensity, Chance of battle by role, double enemy vehicules, Allow enemy VTOL/aeros/turrets, Attached allied units contro, dropship use).


=====================
SCENARIOS IN STRATCON MODE
=====================

* StratCon will generate scenarios based on the mission. If such a scenario happen, it will be displayed in the Briefing room AND the "AtB Campaign State" (as a red square).

* If a scenario appears only in the Briefing tab, then it's a legacy or special scenario (e.g. cache, big battle), and you can ignore it if you wish (you would need to GM-remove it).

* You can also trigger scenario by scouting out the Track. See below.

* Skipping scenarios.
Under integrated or house command, or if the liaison is coming along for the ride, you will lose a VP.
Additionally, if there's an allied facility on the track, the scenario will move towards the nearest allied facility. Sometimes, you need to keep those intact as part of the contract. And if they get captured without a fight, the enemy will put them to use, making subsequent scenarios more difficult.


=======================================
YOUR MISSION AND GOALS IN STRATCON MODE
=======================================

Your Contract Score is no longer the sole metric of your success.

* Under Integrated and House command, you need to keep your VP count positive. Every time you win a scenario which you didn't initiate, that's +1 VP. Every time you lose a scenario you didn't initiate, that's -1 VP. 

* Under Liaison command, you need to keep your VP count positive AND you're responsible for completing the given number of strategic objectives. On defensive contracts, this may include keeping allied facilities intact and under allied control. On offensive contracts, it's usually capturing/destroying facilities or winning specific scenarios. VPs are only affected by scenarios in which a liaison participates.

* Under Independent command, you're only responsible for completing strategic objectives.


=====================
THE STRATCON HEX GRID
=====================

* The hex grid represents the planet you're on. Each (up to) 8x8 section is a "Track".

* Your "base", if you would have such a thing, is abstracted. It does not appear on the Track, though there might be allied Strategic Objectives (places such as a supply or tank depot).

* Unscouted hexes are dark grey. Scouted hexes are light grey. (See below how to scout)

* Allied facilities are displayed as green squares. Hostile facilities as red squares.
* Your forces currently deployed to the track are cyan circles.
* Pending scenarios against hostile forces are concentric red squares.


============
HOW TO SCOUT
============

* To send a Lance to scout, right click on an unscouted (dark gray) hex and select Manage then the Lance you want to send. Scout-role Lances can scout out all surrounding hexes at once.

* You can "cheat" by using GM Mode and right-clicking to reveal it all.

* If your scouts find an Objective, this will immediately create a scenario, though it may be pending for a few days. Note that the scouting itself may also create an encounter with hostiles that wouldn't exist otherwise.

* Every time you send a Lance, to scout or engage a scenario, it will be unavailable (including for repairs) for a duration equal to the "Deployement Period" that's listed in the right pane. If you click on the relevant hex, the pane will show return dates.

* In theory, this should be the same for wounded warriors and salvaged units. As of 49.0, however, they teleport back to base immediately.


=======================
DEPLOYEMENT IN STRATCON MODE
=======================

* First, as of 49.0, the "Deployment Requirements" in the Briefing tab are not implemented in StratCon. However, the Lance count is important, as you will get multiple scn, up to that lance count. Also the assigned Roles are very important in StratCon, as you will see below.

* To deploy in StratCon mode, you must right-click on the relevant Track hex with nested red squares and Manage. Then you can choose which Lance(s) to send, and what reinforcement to send. You can multi-select, and use ctrl-click to unselect. (The "old"  TO&E deployement tool still works, but is mostly kept for GM use; use the Campaign tab one instead.)
 
* Note that when a Lance is deployed, you can right-click on that hex and click on a Lance's checkbox to have it remain in that hex, so that you don't have to spend resources to deploy it there later. Now you have a solid reason for having spare Lances! You can't repair/resupply it, though.

* If under Integrated command, you don't get to choose! The command will choose for you... Otherwise, a Lance is randomly auto-selected for the purpose of computing the OpFor BV, and then you can change it. The exception is when you are scouting or voluntarily creating a battle.

* To send reinforcements, you have several choices:
- You can simply spend a Support Point per Lance (or a Victory Point, in a pinch). SPs are obtained by certain scenarios or holding supply depot Objectives, and leftover Negotiation.
- If a Lance is already on the same hex as the scenario (likely because you've set it to remain there, see above), it can deploy for free.
- A lance in a 'fight' role can be deployed for free. However, on a 8 or less on a 2d6, it will add an extra negative modifier to the scenario. There will be feedback in the Activity Log.
- If your primary Lance leader has leadership, and you have "auxiliary units" available (ones that are not of the same type as the main unit type of the primary force), you can deploy as many auxiliaries as you have leadership points.
- If your primary lance is in a 'defend' role, you can deploy up to 2 * (x + 1) your lance leader's tactics skill in infantry, battle armor squads and minefields. Any "left-over" defensive points are automatically converted into standard minefields.
- Otherwise, you can attempt to deploy a lance anyway. On a 2d6 roll, 2-5 means deployment failure, 6-8 adds extra negative modifier, 9 or more succeeds.


==========================
A NOTE ON OPFOR GENERATION
==========================

The primary opposing forces are usually generated via the "scaled BV" method. This takes into account the difficulty rating and your current CamOps or FMM unit rating. The algorithm is roughly as follows:
- Your unit rating determines the BV floor. This is a percentage from 50-100, inclusive.
- Your difficulty determines the BV ceiling. This a percentage between 80-120, inclusive.
Multiply the above percentages by the BV of your primary force (the lance that gets somewhat-randomly selected when the scn is generated). The resulting BVs are the BV floor and BV ceiling, respectively. Note that, on non-Integrated command, the primary force is a randomly-picked lance from your TO&E, and you then get to choose which lance is actually assigned to the fight. 

1. The scaled opfor receives one lance off the appropriate RAT. If the BV is under the floor, repeat.
2. If the BV is between the floor and ceiling, we roll a 1d100 and compare it to the ratio of current opposition BV / ceiling. If the roll is lower than the ratio, then we stop. Otherwise, go back to 1 and add another lance.

The intention here is that, as the unit gets better, it will face tougher opposition, while giving a bit of a break to starting units. This mechanism may not produce very fair results for very low-BV forces, so there is no expectation of a balanced scenario for four squads of foot rifle infantry. For very high-BV forces, you are likely to see excessive numbers of hostile units.


============================
DESIGNING YOUR OWN SCENARIOS
============================

Use the Scenario Template Editor to put together a scenario template. I recommend using the Briefing Room -> Add Scenario -> Generate From Template mechanism to test the template out. Now, go to your data\scenariomodifiers directory and create (if it doesn't exist already) a file called UserScenarioManifest.xml. It should be formatted exactly like ScenarioManifest. Make sure not to overlap with the ID numbers or file names in ScenarioManifest, as that "may cause unpredictable behavior". Now that you've got your scenario in the UserScenarioManifest file, it'll be added to the pool of scenarios StratCon draws from when generating battles.


===========================================
SPECIFIC STRATCON SCENARIO NOTES AND TRICKS
===========================================

First, note that when there's several objectives listed in the second white box of a scenario write-out, it usually means you have to complete one of those, not all. Also, they usually lead to different outcomes. This is detailed in the post-op objective review, but not in the scenario statement.

* There's too many enemy reinforcements! *
Might be a hostile facility or three on the track. Better do some recon and take them out, or, even better, capture them so you can use them. Or, you might just be having bad luck - each scenario can receive up to three modifiers.

* Ground control*
By default, you control the terrain at the end if you've met the threshold to rout the enemy ground forces.

* Disabling Turrets *
There is no direct sure-fire way to go about disabling turrets. You have two choices:
- Shoot at the buildings and hope to get lucky. Each shot that is over the damage threshold can disable the turret's crew or weapon, which does the trick. However your bot ally will likely keep firing at turret buildings... Also setting turret buildings on fire may "help".
- Eliminate the stated % of the ground enemy forces. Once that's achieved, you're considered to have routed the forces and able to take control of the turrets.

* Base/objective attacks and capture*
That one's a bit tricky. 
- To capture the base you need to *preserve* ALL the turrets. As explained above, your best bet is to NOT shoot at the turrets (though crew/weapon-disabled turrets count as preserved), then rout the indicated % of forces. That's very hard if you have bot allies as they love to shoot and kick buildings; a trick is to give all allied bots all non-turret enemies as strategic targets and set their agression level lower.
- To destroy the base, you need to destroy or disable ALL Turrets. You don't actually need to rout the ground forces.
- To recon the base, you need to scan all turrets. You might want to tweak allied bots as above. Jump jets are a must. Note that if you get a base Recon scenario, you're not bound to stick to recon and can still go for a capture or destroy. You might consider the scenario a failure, though, I guess.

* Scouting *
Some scenarios ask you to spot/attack a unit to scout it out. Remember that MHQ will not keep a tally for you. So make sure to make a list first and check those you've so spotted.

* Fleeing units *
MHQ considers all fled units as withdrawn, even if they fled when mint as part of their bot routine. In scenarios where you must destroy a unit (e.g. assassinate or convoy attack) you should manually take note of units that have actually fled and those that were destroyed and double-check upon scenario end.
```

**Comment by Tecmes** [comment]
Updated with answers from NickAragua in the forums.

**Comment by Windchild292** [comment]
@NickAragua

**Comment by Windchild292** [comment]
Merged as part of 08aa9c7913bf55a8e30862cd048674b9e0d1c461
